src/main/java/calculator/StringSplitter.java

```diff
@@ -0,0 +1,24 @@
+package calculator;
+
+public class StringSplitter {
```

Comment:
생성자의 접근 제한자를 private으로 설정하면 유틸리티 클래스가 인스턴스화되는 것을 방지할 수 있습니다. 

```suggestion
    private StringSplitter() {
    }
```

Reply:
유틸리티 클래스를 명확히 표현하기 위해 접근 제한자 부분까지 신경써야 한다는 점을 잊고 있었네요 ! 꼼꼼한 피드백 감사드려요!

Reply:
~~반영이 안되어있네요!~~

Reply:
아래와 같이 생성자를 private로 반영했어요 !
```java
private StringSplitter() {
        throw new UnsupportedOperationException("StringSplitter is a utility class and cannot be instantiated.");
    }
```

Reply:
아하 제가 코드 확인을 못한 것 같습니다. 잘 반영해주셨네요 👍